website/videoBlog/2024-02-10-Data-Ingestion-to-Visualization-Hudi-MinIO-StarRocks-HiveMetaStore-Apache-SuperSet-Hands-on-Guide.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: "Data Ingestion to Visualization: Hudi + MinIO + StarRocks + HiveMetaStore + Apache SuperSet Hands on Guide"
+last_modified_at: 2024-02-21T16:54:38.964863-07:00
+authors:
+- name: Soumil Shah
+category: blog
+image: /assets/images/video_blogs/2024-02-10-Data-Ingestion-to-Visualization-Hudi-MinIO-StarRocks-HiveMetaStore-Apache-SuperSet-Hands-on-Guide.png
+navigate: "https://www.youtube.com/watch?v=JkKBzrQTKx0"
+tags:
+- guide
+- beginner
+- apache hudi
+- starrocks
+- hive metastore
```

Comment:
Add `apache hive` tag? And also add `hive metastore` to other videos -  I see http://hudi.apache.org/videos/tags/apache-hive  for other relevant videos where there is a hive metastore component.

Reply:
Please address feedback above!

Reply:
Add `apache hive` tag here and `hive metastore` tags for all the blogs http://hudi.apache.org/videos/tags/apache-hive . I only see 2/5 blogs amended for this tag - `hive metastore`.

Reply:
added to all the blogs